completion/available/ngrok.completion.bash

```diff
@@ -0,0 +1,45 @@
+# shellcheck shell=bash
+
+if _binary_exists ngrok; then
```

Comment:
no need for this check, as the completion is not using the binary itself

Reply:
@NoahGorny Done